corpus/config_db.json

```diff
@@ -1,10 +1,63 @@
+      "name": "CompSoc",
+      "slug": "compsoc",
+      "what_we_do": "Intelligence deals with fields such as Machine Learning, Artificial Intelligence, Deep Learning, Computer Vision, NLP.\r\nSystems is related to fields involving Operating Systems, Parallel Computing, Network Security, Distributed Systems .\r\nCore Computer Science involves Algorithms, Cryptography, Databases, Game Theory, Web & Software Development ."
+    },
+    "model": "config.sig",
+    "pk": 1
+  },
+  {
+    "fields": {
+      "about": "The NITK Diode SIG is a combination of the Circuits and Systems (CAS) Society, Signal Processing Society (SPS) and the Industrial Applications Society (IAS).\r\nThe SIG consists of a team of students with a strong interest in the field of Electronics and Electrical Science, and works on conceiving and pioneering solutions to fundamental and applied problems in the fields of robotics, microcontrollers and signal processing.\r\nIEEE NITK inaugurated the IEEE Industrial Applications Society student chapter in the year 2019. This year we saw a steady increase in the number of activities and events organized by the student chapter.",
+      "name": "Diode",
+      "slug": "diode",
+      "what_we_do": "For 2019-20 IAS saw one project on a battery based solar powered inverter being completed under its wing. To enrich its members with knowledge in the fields of power electronics, the chapter held 3 workshop sessions. Additionally, in collaboration with the Department of Electrical and Electronics Engineering, the IAS student chapter organized a national conference PETPES 2019."
+    },
+    "model": "config.sig",
+    "pk": 2
+  },
+  {
+    "fields": {
+      "about": "Piston is a SIG dedicated to the engineering applications of Mechanical, Automation, Civil, Chemical and Material Sciences. The SIG functions as a platform for students interested in learning and advancing specific areas of knowledge and technology. The SIG members co-operate, communicate, and organise meets, talks and conferences to discuss the latest technological advancements and applications.",
+      "name": "Piston",
+      "slug": "piston",
+      "what_we_do": "The SIG specializes on a lot of areas like Aerospace Engineering, Automobile Engineering, Bio-Mechanical Applications, Chemical and Thermal Engineering, Heat and Renewable Energy Engineering, Robotics, Controls &amp; Mechatronics and Structures &amp; Materials."
+    },
```

Comment:
Minor deletion in Piston "about" and "what_we_do" field : 
```
{
      "about": "Piston is a SIG dedicated to the engineering applications of Mechanical, Civil, Chemical and Material Sciences. The SIG functions as a platform for students interested in learning and advancing specific areas of knowledge and technology. The SIG members co-operate, communicate, and organise meets, talks and conferences to discuss the latest technological advancements and applications.",
      "name": "Piston",
      "slug": "piston",
      "what_we_do": "The SIG specializes on a lot of areas like Aerospace Engineering, Automobile Engineering, Chemical and Thermal Engineering, Heat and Renewable Energy Engineering, Robotics, Controls &amp; Mechatronics and Structures &amp; Materials."
    },
```

Removed automation from 'about' and bio-mechanical applications from 'what we do'.

Reply:
done, resolved